spec/features/visitor_can_view_items_in_cart_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.